src/main/resources/application.yml

```diff
@@ -0,0 +1,13 @@
+spring:
+  application:
+    name: userProvisioning-service
```

Comment:
Don't forget to add the `@EnableEurekaClient` annotation on the class that holds the `main()` method.

Reply:
Strange, it registers with the server without the annotation. Seems that is optional, only to be used when you know for sure that is Eureka discovery client that you want to use.